docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -131,10 +131,12 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
 are denoted `class 0` and `class 1`, then the value of `class_score` is always 
 non-negative and its definition is:
 
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+`class_score (class 0)` 
+stem:[= 0.5 / (1.0 - k) * probability(class 0)]
```

Comment:
Should we move the first bit into the equation too, so it's got the same formatting?
```suggestion
stem:[\text{class_score(class 0)} = 0.5 / (1.0 - k) * probability(class 0)]
```